app/utils/capture-exception.ts

```diff
@@ -1,15 +1,34 @@
+import { ErrorObject } from 'jsonapi-typescript';
+import { ErrorDocument } from 'osf-api';
 
 // Raven is defined only in prod builds
 declare const Raven: undefined | {
-    captureException(e: Error): void;
+    captureException(e: ErrorDocument | Error, extra: object): void;
 };
 
+export function getApiError(error: ErrorDocument): ErrorObject|undefined {
+    let apiError;
+    if (Array.isArray(error.errors) && error.errors.length &&
+        typeof error.errors[0].detail === 'string') {
+        [apiError] = error.errors;
```

Comment:
what if there are multiple errors? is that a thing that can happen with our api?

(if there are multiple errors, we include all of them when we send to sentry, at least, and maybe consider concatting in the toast message, as well)

Reply:
It seems [possible](https://github.com/CenterForOpenScience/osf.io/blob/develop/api/base/exceptions.py#L51) to multiple errors, but I have not seen it on FE. I will add multiple error support functionality.